extensions/cornerstone/src/getCustomizationModule.ts

```diff
@@ -0,0 +1,37 @@
+import { Enums } from '@cornerstonejs/tools';
+import { toolNames } from './initCornerstoneTools';
+
+const tools = {
+  active: [
+    {
+      toolName: toolNames.WindowLevel,
+      bindings: [{ mouseButton: Enums.MouseBindings.Primary }],
+    },
+    {
+      toolName: toolNames.Pan,
+      bindings: [{ mouseButton: Enums.MouseBindings.Auxiliary }],
+    },
+    {
+      toolName: toolNames.Zoom,
+      bindings: [{ mouseButton: Enums.MouseBindings.Secondary }],
+    },
+    { toolName: toolNames.StackScrollMouseWheel, bindings: [] },
+  ],
+  enabled: [{ toolName: toolNames.SegmentationDisplay }],
```

Comment:
Thank you for moving this - we may want to eventually tweak these to be easier to modify, but it looks good for now.